### DISCUSSION TITLE
PerfView not showing values in 'Exc' columns as expected?

### DISCUSSION THREAD

**Discussion by kiranongithub** [microsoft/perfview#1567] [Q&A]
I am just trying to understand PerfView basics.

**Question**: I was expecting PerfView to show data in all of he "Exc" columns for the `FindPrimeNumber` method since this method does not call any other method. But I see `0` in all these `Exc` columns. Why is that?

I have the following class in a .NET Core 6 application:
```c#
using System;

namespace MyApp // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");
            Slow();
            Console.WriteLine("Done");
        }

        static void Slow()
        {
            long nthPrime = FindPrimeNumber(10000000); //set higher value for more time
        }

        static long FindPrimeNumber(int n)
        {
            int count = 0;
            long a = 2;
            while (count<n)
            {
                long b = 2;
                int prime = 1;// to check if found a prime
                while (b * b <= a)
                {
                    if (a % b == 0)
                    {
                        prime = 0;
                        break;
                    }
                    b++;
                }
                if (prime > 0)
                {
                    count++;
                }
                a++;
            }
            return (--a);
        }
    }
}
```

I did a `dotnet-trace` of this app in a docker container (fyi, I did NOT set any COMPlus environments variables..was not sure if that is required).  Following is a snapshot of the trace.

![image](https://user-images.githubusercontent.com/68080545/153714293-3b07ff92-18fb-44dd-9f83-e0fb3b8e859d.png)

**Comment by brianrob** [comment]
@kiranongithub, if you scroll to the very top of the view, you should see that there are going to be one or two nodes that have non-zero exclusive time.  This is because one of these two nodes are always at the top of the stack.  The nodes represent unmanaged code time, and CPU time.  If you want to have the time attributed to the functions themselves, you can choose these nodes, and fold them (right click and select Folding --> Fold item).  This should give you what you're looking for.
